src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const correctRobor = {};
+
+  for (const key in robot) {
+    if (correctRobor.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    correctRobor[robot[key]] = key;
```

Comment:
you repeat robot[key] 2 times here, better create variable with this value

Reply:
ok:)